modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -510,15 +510,51 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
     using ParamType = std::pair<InferenceEngine::TensorDesc,
                                 InferenceEngine::ParamMap>;
 
-    ParamType* blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
-    if (blob_params == nullptr) {
+    ParamType* frame_blob_params = cv::util::any_cast<ParamType>(&any_blob_params);
+    if (frame_blob_params == nullptr) {
         GAPI_Assert(false && "Incorrect type of blobParams: "
                               "expected std::pair<InferenceEngine::TensorDesc,"
                                                  "InferenceEngine::ParamMap>");
     }
 
-    return ctx.uu.rctx->CreateBlob(blob_params->first,
-                                   blob_params->second);
+    if (ctx.uu.params.device_id.find("GPU") != std::string::npos) {
+        GAPI_LOG_DEBUG(nullptr, "Extract remote blob for GPU DX11 device_id: " <<
+                                ctx.uu.params.device_id);
+        // despite of layout, blob dimensions always follow in N,C,H,W order
+        const auto& dims = frame_blob_params->first.getDims();
+        InferenceEngine::TensorDesc desc(InferenceEngine::Precision::U8,
+                                         {1, 1, dims[2], dims[3]},
+                                         InferenceEngine::Layout::NHWC);
+
+
+        InferenceEngine::ParamMap blobParams = {
+                           {"SHARED_MEM_TYPE", "VA_SURFACE"},
+                           {"DEV_OBJECT_HANDLE", frame_blob_params->second["DEV_OBJECT_HANDLE"]},
+                           {"VA_PLANE", uint32_t(0)}};
+        // TODO NV12 surface supported only
+        if (static_cast<InferenceEngine::ColorFormat>(frame_blob_params->second["COLOR_FORMAT"]) ==
+            InferenceEngine::ColorFormat::NV12) {
+            InferenceEngine::Blob::Ptr y_blob =
+                           std::dynamic_pointer_cast<InferenceEngine::Blob>(
```

Comment:
Can this be done with some make_..._blob functions? For interest.

Reply:
yes it is: 524 - 550 lines related to `make_shared_nv12_blob` API function and to declare that function we need to include specific OV include file with this make* API but this header file has a required dependency from Open**CL** -HPP package which is not a part of open**CV** and should be not according to license type. So all make_* functions from OV simply unwrapped from its definition and copied here